src/app/groups/groups.component.html

```diff
@@ -1,28 +1,31 @@
-<div class="container-fluid p-0">
+<div class="container-fluid p-0 h-100">
+    <app-modal-add-group></app-modal-add-group>
     <div class="container">
         <h1 class="mb-5 font-weight-bold ml-2">Select group</h1>
         <div class="row align-content-space-evenly justify-content-around justify-content-sm-start m-0">
             <div class="col-12 col-md-4 col-lg-3 mb-4 p-2" *ngFor="let group of groups">
                 <div class="card">
-                    <div class="img-wrapper p-2">
-                        <img class="card-img-top w-100" src="../../assets/{{group.img}}" alt="Group image">
-                    </div>
+                    <a [routerLink]="['/groups', group.id]">
+                        <div class="img-wrapper p-2">
+                            <img class="card-img-top w-100" src="../../assets/{{group.img}}" alt="Group image">
```

Comment:
Would it be better to use absolute Url (`src="/assets/{{group.img}}"`)?

Reply:
You're right! Fixed